### DISCUSSION TITLE
Enable the weapons on weaponised vehicles

### DISCUSSION THREAD

**Discussion by theblaziken** [YimMenu/YimMenu#2260] [Q&A]
![IMG_6062](https://github.com/YimMenu/YimMenu/assets/147301459/d5b6cb1d-6168-41b6-9602-b01697824dc7)
How do I get the cannons working on these vehicles?  As you can see in the image above, even the crosshair doesn't appear on my Khanjali. I'm facing the same problem on the Rhino and the Raiju jet as well

**Comment by binded-zz** [comment]
stupid question did you change seats  three's a drivers seat and a shooters seat 

you can goto LSC and sometimes change the weapons that are loaded and or upgrade 
then three's in the fun tab you can change the rocket and cannons explosions types 
i play with a controller so i push the i think dpad right to get in the seat  and most of the time you can change i use Y  what ever that is on the kb

you might want to go and look at your kb bindings  to see what does what

**Reply by theblaziken** [reply]
Nope, I'm the main drivers seat, and all my kb bindings are the exact same as I know them. If I dont inject the menu, it works as intended

**Reply by binded-zz** [reply]
yeah idk it works for me just fine

**Comment by lonelybud** [comment]
using passive mod?

**Reply by theblaziken** [reply]
Nah
